main.py

```diff
@@ -0,0 +1,32 @@
+import argparse
+from vid2persona import init
+from vid2persona.pipeline import vlm
+
+def workflow(args):
+    traits = vlm.get_traits(
+        args.gcp_project_id, 
+        args.gcp_project_location, 
+        args.target_movie_clip, 
+        args.prompt_tpl_path
+    )
+    print(traits)
+
+def sanity_check(args):
```

Comment:
`validate_inputs` might be a better name here. Also WDYT about doing this using a decorator, responsible for validating the necessary inputs? We could decorate the `workflow()` function then.

Reply:
Never thought about decorator! Good suggestion!